graphql-dgs-subscriptions-websockets/src/test/kotlin/com/netflix/graphql/dgs/subscriptions/websockets/WebsocketGraphQLWSProtocolHandlerTest.kt

```diff
@@ -228,7 +229,7 @@ class WebsocketGraphQLWSProtocolHandlerTest {
 
         assertThat(dgsWebsocketHandler.sessions.size).isEqualTo(currentNrOfSessions + 1)
 
-        val returnMessage = jacksonObjectMapper().readValue<OperationMessage>(slot.captured.asBytes())
```

Comment:
Is this usage  problematic?

Reply:
Not in and of itself, it just felt more natural to put the ObjectMapper instance in a field since it's used in more than one test.